### DISCUSSION TITLE
The classification of OSMR(Offsec).

### DISCUSSION THREAD

**Discussion by R-yu-2** [PaulJerimy/SecCertRoadmapHTML#112] [General]
OSMR is currently classified as "Penetration Testing", but isn't it classified as "Exploitation" in terms of content?
It is also listed as "Exploit Development" on Offsec's website.

https://www.offsec.com/courses/exp-312/

**Comment by gogooo121** [comment]
it's exploitation + penetration testing course so i do believe it can be a bit wider to fit both classifications.

Also the rating of OSWA and OSDA should be higher, both Certifications are considered intermediate and for OSDA its really goes deep into threat hunting  and incidents identification.
